tests/util/benchmark_cost.py

```diff
@@ -113,7 +113,7 @@ def benchmark_all_operators():
 
     for i in range(0, 1000):
         private_key: PrivateKey = master_sk_to_wallet_sk(secret_key, uint32(i))
-        public_key = private_key.public_key()
+        public_key = private_key.get_g1()
```

Comment:
this seems like an old typo, it was introduced in the first commit of this file, Tue Mar 24 23:03:31 2020